### PR TITLE
Add figures to methods and edit text

### LINKED ISSUE #1026: Overview figure for methods manuscript
I attempted to sketch an overview figure to summarize some of the automation in this project and how GitHub is used.  It's pretty complex.  That makes it hard to draw an accurate and simple figure but also makes such a figure valuable to readers.

Here's my first draft:
COVID-19 methods overview v1

It's in Google Drawings so I can give others access if they want to play around with it.  @rando2 does this make sense?  Do we want to add an overview figure?

_Edit:_ One aspect that is confusing in this version is that "Files" and "Outputs" are separate columns.  In reality, many of the files tracked on these branches are outputs from a GitHub Actions workflow that are committed to the branch.

### PR DESCRIPTION
**Pull request by agitter**
Closes #1026
Add figures from #944 and #1034

There are several changes to automated statistics, new figures, and a manual reference that will need to be checked carefully in the artifacts before merging.  I started some text edits, most notably shortening the abstract, but did not yet do my final proofreading.

I'm proposing we remove the references to individual manuscripts that haven't hit the preprint stage yet if we need to save space:
> These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing disease transmission and outcomes [@individual-inequality].

They took up a few lines of references in the prior DISCO submission and aren't that informative:
image

### CODE REVIEW

**Comment by agitter** [comment]
I quickly checked the PDF artifact from build 1.0.4488:
- Figures 9, 10, and 11 look great
- I didn't see any problem with the automated stats in my first pass
- Manual reference 1542 worked

File: content/60.methods.md [hunk 1]
```diff
@@ -3,30 +3,16 @@
...
-Through this effort, we organized over 50 scientists from a range of backgrounds who evaluated over 1,000 sources and developed seven literature reviews.
-This project illustrates that Manubot is an adaptable workflow that can handle even an extreme volume and demonstrates that, despite the back-end technical complexity of Manubot, non-technical contributors can participate in Manubot projects.
-While many efforts from the computational community have focused on mining COVID-19 literature, this implementation illustrates the power of open publishing to organize people to aggregate and disseminate information in response to an evolving crisis.
-Applying this approach allowed us to develop a community that occupies a unique niche within the COVID-19 research space.
-
-{% if individual is defined and individual == "methods" and format is defined and format == "tex"-%}
-### CCS CONCEPTS
-<!--To Do: @mprobson, do you want to take the lead on this?-->
-{%- endif %}
+The multifaceted nature of COVID-19 demands a multidisciplinary approach, but the urgency of the crisis combined with the need for social distancing measures present unique challenges to collaborative science.
```
Comment (rando2) [inline content/60.methods.md:6]: Way more succinct abstract, looks great to me!

File: content/60.methods.md [hunk 2]
```diff
@@ -191,34 +186,38 @@ Manubot users can write in the simple markdown format, render the manuscript in
...
 
 In order to make the project more accessible to individuals from a range of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
 We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request^[https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md and https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
 Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.
 Articles were classified as _diagnostic_, _therapeutic_, or _other_, with an associated template developed to guide the review of papers and preprints in each category.
-A total of 273 new paper issues had been opened as of April 30, 2021.<!--To Do: automate-->
+A total of 273 new paper issues had been opened as of April 30, 2021.<!--To Do: automate (currently have open and closed issues but not paper issues) -->
```
Comment (rando2) [inline content/60.methods.md:214]:
```suggestion
A total of 285 new paper issues had been opened as of September 13, 2021.<!--To Do: automate (currently have open and closed issues but not paper issues) -->
```

File: content/60.methods.md [hunk 3]
```diff
@@ -191,34 +186,38 @@ Manubot users can write in the simple markdown format, render the manuscript in
...
+These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing disease transmission and outcomes [@individual-inequality]. <!-- To Do: these self-references take up a lot of space in the PDF, should we leave the text and remove the citations? -->
```
Comment (rando2) [inline content/60.methods.md:220]:
```suggestion
These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines, and diagnostic technologies), and biological and social factors influencing disease transmission and outcomes.
```

**Review by rando2 — APPROVED** [review]
Screen Shot 2021-09-14 at 10 25 22 AM

The new json variables seem to be integrating in LaTex as well 🎉 (I was checking this when I saw your update!)

**Comment by rando2** [comment]
I also looked through the PDF and it looked good to me. Looks like just around 10 pgs...

**Comment by agitter** [comment]
Thanks for merging.  I'll work on porting this to Overleaf in a couple hours for finishing touches.

**Comment by rando2** [comment]
> Thanks for merging. I'll work on porting this to Overleaf in a couple hours for finishing touches.

Awesome! I've been making some superficial changes to the formatting in the .tex file, so I'll post that here around when your class ends!

**Comment by rando2** [comment]
[methods.bib.txt](https://github.com/greenelab/covid19-review/files/7164597/methods.bib.txt)
[methods.tex.txt](https://github.com/greenelab/covid19-review/files/7164598/methods.tex.txt)

These are modified to:
- Display "COVID-19 Review Consortium" literally instead of trying to make it into firstname/lastname format
- Fix the display of the table (which didn't convert correctly)
- Number the author affiliations
- Move the COI to the end and trim the text to get it to fit in one column
- Adjust some particularly long/weird footnotes

**Comment by agitter** [comment]
I set up a new Overleaf project with these steps:

- Create empty project
- Copy `cc-by.pdf` `ceurart.cls` `ceur-ws-logo.pdf` from old project
- Download and add https://github.com/greenelab/covid19-review/blob/7f4c4afdd75240b9e492156eb2ebd3c5f30f1646/methods.bib
- Download and add https://github.com/greenelab/covid19-review/blob/7f4c4afdd75240b9e492156eb2ebd3c5f30f1646/methods-manuscript.tex
- Set main document to methods-manuscript.tex
- Label project
- Download and add https://github.com/greenelab/covid19-review/files/7164597/methods.bib.txt and https://github.com/greenelab/covid19-review/files/7164598/methods.tex.txt from above
- Upload to Overleaf project
- Label project

**Pull request merged**